README.md

```diff
@@ -74,3 +74,25 @@ ctx := context.Context{
 
 unleash.IsEnabled("someToggle", unleash.WithContext(ctx))
 ```
+
+## Development
+
+Requirements:
+* make
+* golint (go get -u golang.org/x/lint/golint)
+
+Run tests:
+
+    make 
+    
+Run lint check:
+
+    make lint
+    
+Run code-style checks:(currently failing)
```

Comment:
It would be nice if all the checks were passing before merging this, but I haven't had time lately to fix this so I can just merge it for now and we can create issues for fixing these things.

Reply:
I think a separate issue for the actual fixes would be cleaner :)